k8s_support.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Kubernetes support"
+nav_order: 9
+description: "Deploy FalkorDB to Kubernetes."
+---
+
+## Kubernetes support for FalkorDB
+
+FalkorDB can be deployed on Kubernetes using Helm charts and Docker images. This guide will walk you through the process.
+
+### Prerequisites
+
+Before you begin, make sure you have Helm installed on your Kubernetes cluster.
+
+To deploy FalkorDB to Kubernetes we need to use:
+
+* [Helm charts](https://bitnami.com/stack/redis/helm)
+* [Docker image](https://hub.docker.com/r/falkordb/falkordb)
+
+And follow this steps:
+
+### Step 1: Create a `values.yaml` File
+
+1.Create a values.yaml file with the following content:
+
+```yaml
+image:
+  registry: docker.io
+  repository: falkordb/falkordb
+  tag: "4.0"
```

Comment:
There's no way we will remember to update this every time we release a new version

Reply:
it is example